src/test/a11y/tests/admin-routes.test.ts

```diff
@@ -83,4 +83,180 @@ describe('Accessibility - Admin Routes', () => {
             testAccessibility(route.path, route.parameter, route.postMethod, route.postBody);
         });
     });
+
+    describe('Page with Errors', () => {
+        describe('A-Z Search Page', () => {
```

Comment:
This is a public route as u do not need to sign in to be able to access the page

Reply:
Modified test. Thanks Kian.